packages/console/src/pages/TenantSettings/TenantMembers/InviteEmailsInput/hooks.ts

```diff
@@ -44,15 +44,16 @@ const useEmailInputUtils = () => {
       const validEmails = new Set<string>();
 
       const existingMemberEmails = new Set<string>(
-        existingMembers.map(({ primaryEmail }) => primaryEmail ?? '').filter(Boolean)
+        existingMembers.map(({ primaryEmail }) => primaryEmail?.toLowerCase() ?? '').filter(Boolean)
```

Comment:
why don't we lowercase all emails in the beginning?

Reply:
You mean lower case the email on submit?